components/greetings.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [name, setName] = useState('')
+
+  const updateName = (event) => {
+    if (event.target.value) {
+      setName(`It's nice to meet you ${event.target.value}`)
```

Comment:
You're storing an entire greeting in state here not just the name. Since the name is the only dynamic part that's all you should store and let the rest of it live in the element itself.